server/aws-lsp-identity/src/language-server/identityServer.ts

```diff
@@ -55,6 +56,7 @@ export class IdentityServer extends ServerBase {
             ssoCache,
             autoRefresher,
             { showUrl, showMessageRequest, showProgress },
+            this.features.identityManagement.sendGetMfaCode,
```

Comment:
For client->server requests/notifications wrap them and pass that in to IdentityService in the 4th parameter (see line 58 above for examples).